pkgs/tools/networking/bitmask-vpn/default.nix

```diff
@@ -25,12 +27,11 @@
 let
   version = "0.24.8";
 
-  src = fetchFromGitLab {
```

Comment:
It's still gitlab. Why's this change? (fetchFromGitLab also has leaveDotGit)

Reply:
You're right, I must have misread the fetchers section of the reference manual and thought that leaveDotGit required using fetchgit. What the docs actually say is that if leaveDotGit is true then the platform-specific fetcher will revert to using fetchgit automatically but that doesn't require any other change. Thanks for catching that, I'm pushing a new version switching back to fetchFromGitLab.